pluto_build/02_build.sh

```diff
@@ -89,16 +89,17 @@ psql $BUILD_ENGINE -f sql/landuse.sql
 psql $BUILD_ENGINE -c "VACUUM ANALYZE pluto;"
 
 echo 'Flagging tax lots within the FEMA floodplain'
+psql $BUILD_ENGINE -f sql/latlong.sql
+psql $BUILD_ENGINE -f sql/update_empty_coord.sql
 psql $BUILD_ENGINE -f sql/flood_flag.sql
 psql $BUILD_ENGINE -c "VACUUM ANALYZE pluto;"
 
 echo 'Assigning political values with spatial join'
 # clean up numeric fields
```

Comment:
This comment `# clean up numeric fields` should be between `psql $BUILD_ENGINE -f sql/spatialjoins.sql` and `psql $BUILD_ENGINE -f sql/numericfields_geomfields.sql` See original file in main branch.